iFixit/react-commerce
# Add featured collection section

This PR closes #29  

## QA

1. Visit the Vercel PR preview
2. Visit the parts collection page
3. Verify that the featured collection section is rendered with content from Strapi
